spine-unity/Assets/spine-unity/SkeletonAnimator.cs

```diff
@@ -202,4 +204,14 @@ public enum MixMode { AlwaysMix, MixNext, SpineStyle }
 			_UpdateComplete(this);
 		}
 	}
+
+	private int GetAnimationClipNameHashCode(AnimationClip clip) {
+		int clipNameHashCode;
+		if (!clipNameHashCodeTable.TryGetValue(clip, out clipNameHashCode)) {
```

Comment:
Is a method call and hash map look up really faster than computing a string hashcode? I would expect string internally caches the hash code (Java does, FWIW).